backend/molgenis-emx2-webapi/src/test/java/org.molgenis.emx2.web/WebApiSmokeTests.java

```diff
@@ -252,6 +252,15 @@ public void testReports() throws IOException {
             .asString();
     Object result = objectMapper.readValue(jsonResults, Object.class);
     assertTrue(result.toString().contains("pooky"));
+
+    // test report using json objects
+    jsonResults =
```

Comment:
does it also work if you run it with multiple reports? I thought that was the issue?
I.e.  ?id=report1&id=report2 or ?id=report1,report2.
Is there a test for that?

Reply:
Ug you're right I broke that again